app/scripts/controllers/preferences-controller.ts

```diff
@@ -17,11 +25,90 @@ const testNetworks = {
   [CHAIN_IDS.LINEA_SEPOLIA]: true,
 };
 
+type PreferencesControllerOptions = {
+  networkConfigurations?: Record<string, { chainId: Hex }>;
+  initState?: Partial<PreferencesControllerState>;
+  initLangCode?: string;
+  // TODO: Replace `any` with type
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  network?: any;
+  // TODO: Replace `any` with type
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  messenger?: any;
+};
+
+type Preferences = {
+  autoLockTimeLimit?: number;
+  showExtensionInFullSizeView: boolean;
+  showFiatInTestnets: boolean;
+  showTestNetworks: boolean;
+  smartTransactionsOptInStatus: boolean | null;
+  useNativeCurrencyAsPrimaryCurrency: boolean;
+  hideZeroBalanceTokens: boolean;
+  petnamesEnabled: boolean;
+  redesignedConfirmationsEnabled: boolean;
+  redesignedTransactionsEnabled: boolean;
+  featureNotificationsEnabled: boolean;
+  isRedesignedConfirmationsDeveloperEnabled: boolean;
+  showConfirmationAdvancedDetails: boolean;
+};
+
+export type PreferencesControllerState = {
+  selectedAddress: string;
+  useBlockie: boolean;
+  useNonceField: boolean;
+  usePhishDetect: boolean;
+  dismissSeedBackUpReminder: boolean;
+  useMultiAccountBalanceChecker: boolean;
+  useSafeChainsListValidation: boolean;
+  useTokenDetection: boolean;
+  useNftDetection: boolean;
+  use4ByteResolution: boolean;
+  useCurrencyRateCheck: boolean;
+  useRequestQueue: boolean;
+  openSeaEnabled: boolean;
+  securityAlertsEnabled: boolean;
+  watchEthereumAccountEnabled: boolean;
+  bitcoinSupportEnabled: boolean;
+  bitcoinTestnetSupportEnabled: boolean;
+  addSnapAccountEnabled: boolean;
+  advancedGasFee: Record<string, Record<string, string>>;
+  featureFlags: Record<string, boolean>;
+  incomingTransactionsPreferences: Record<number, boolean>;
+  knownMethodData: Record<string, string>;
+  currentLocale: string;
+  identities: Record<string, AccountIdentityEntry>;
+  lostIdentities: Record<string, object>;
+  forgottenPassword: boolean;
+  preferences: Preferences;
+  ipfsGateway: string;
+  isIpfsGatewayEnabled: boolean;
+  useAddressBarEnsResolution: boolean;
+  ledgerTransportType: LedgerTransportTypes;
+  snapRegistryList: Record<string, object>;
+  theme: ThemeType;
+  snapsAddSnapAccountModalDismissed: boolean;
+  useExternalNameSources: boolean;
+  useTransactionSimulations: boolean;
+  enableMV3TimestampSave: boolean;
+  useExternalServices: boolean;
+  textDirection?: string;
+};
+
 export default class PreferencesController {
+  store: ObservableStore<PreferencesControllerState>;
+
+  // TODO: Replace `any` with type
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private messagingSystem: any;
+
+  // TODO: Replace `any` with type
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private network: any;
```

Comment:
Same as the below comment if we remove the network property from the constructor options. we can remove this private variable